### DISCUSSION TITLE
Is the free tier supposed to be very slow?

### DISCUSSION THREAD

**Discussion by tonychandesign** [supabase/supabase#13550] [Questions]
Hello!  I'm trying out supabase with a sideproject but noticed that it's extremely slow (5 seconds+ for all requests).  I'm not sure if the issue is with the current stack I'm using or if it's intended behavior for the free tier so I figured I check here

for context- I'm using prisma, nextJS 13, and next-auth 

Here is an example call I'm making to create an item on the DB.  This snippet is apart of a fetch API within the server side of NextJS and it's being called by a client side component.

```
const randomIntFromInterval = (min: number, max: number) => {
  return Math.floor(Math.random() * (max - min + 1) + min);
};
const handler = async (
  req: NextApiRequest,
  res: NextApiResponse<CreateCanvas>
) => {
  try {
    const user = await getServerSession(req, res, authOptions).then(
      (res) => res?.user
    );
    if (!user) {
      return res.status(401).json({
        error: "Unauthorized to perform this action.",
        createdCanvas: null,
      });
    }
    const createdCanvas = await db.canvas.create({
      data: {
        title: "Untitled",
        ownerId: user.id,
        shared: false,
        id: canvasId,
        bgColor: randomColor,
      },
    });
    return res.status(200).json({ error: null, createdCanvas });
};
export default withMethods(["POST"], handler);
```

I constantly clear out the entire DB so there's typically less than 10 items in it but it still usually takes more than 5 seconds per REST call.  I'm also wondering if this is a nextJS 13 bug but haven't seen anyone mention it in the forums.  I'm wondering if it's slow because I'm making the requests from localhost?  I know that nextJS is a bit slow when in dev mode and it requires building the app for it to be performance so I'm wondering if it's the same thing?  I haven't tried deploying since I'm nowhere close to complete but figured I start the discussion here to get more context.  It's just surprising that it take anything more than half a second since it's just creating a single line item without any complex querying.

Any help/comments would be greatly appreciated- I'd love to use supabase for future projects since the web UI is super intuitive and the real time DB features look interesting :D

<img width="505" alt="image" src="https://user-images.githubusercontent.com/22465526/230760571-802f88cd-790f-4d42-b809-247df0e5c145.png">

**Comment by iuliiast** [comment]
I'm on the free plan and my queries take less than a second on average, definitely not 5 seconds.
Stack: Next 13 and Supabase with their Auth.

**Comment by ngamsomset** [comment]
I have the same issue. Took 5 seconds to load.

**Reply by GaryAustin1** [reply]
You probably should start a new post here or a Discord topic on what you are up to and what takes 5 seconds.  A free instance performs about the same as an entry Pro instance.  Normal DB requests 100-200msec latency thru the REST API from what I've seen (in the US) and almost all of that is network time.

**Comment by arshadkhan-ai** [comment]
I am using nest js and facing the same slowness issue. I was using the  postgres on vps.  It was working good there. Now i migrate to supabase but its taking more time in response.

**Reply by GaryAustin1** [reply]
There is a management API issue right now...
https://status.supabase.com/incidents/p78yv2qzjm7d
